documentation/en/chain-indexer-overview-for-operators.md

```diff
@@ -0,0 +1,415 @@
+# ChainIndexer Documentation for Operators <!-- omit in toc -->
+
+- [Introduction](#introduction)
+- [ChainIndexer Config](#chainindexer-config)
+  - [Enablement](#enablement)
+  - [Garbage Collection](#garbage-collection)
+    - [Recommendations](#recommendations)
+  - [Removed Options](#removed-options)
+- [Upgrade](#upgrade)
+  - [Preparation](#preparation)
+  - [Upgrade when using existing `LOTUS_PATH` chain state](#upgrade-when-using-existing-lotus_path-chain-state)
+    - [Part 1: Create a backfilled ChainIndexer `chainindex.db`](#part-1-create-a-backfilled-chainindexer-chainindexdb)
+    - [Part 2: Create a copyable `chainindex.db`](#part-2-create-a-copyable-chainindexdb)
+    - [Part 3: Update other nodes](#part-3-update-other-nodes)
+    - [Part 4: Cleanup](#part-4-cleanup)
+  - [Upgrade when importing chain state from a snapshot](#upgrade-when-importing-chain-state-from-a-snapshot)
+- [Backfill](#backfill)
+  - [Backfill Timing](#backfill-timing)
+  - [Backfill Disk Space Requirements](#backfill-disk-space-requirements)
+  - [`lotus-shed chainindex validate-backfill` CLI tool](#lotus-shed-chainindex-validate-backfill-cli-tool)
+    - [Usage](#usage)
+- [Regular Checks](#regular-checks)
+- [Downgrade Steps](#downgrade-steps)
+- [Terminology](#terminology)
+  - [Previous Indexing System](#previous-indexing-system)
+  - [ChainIndexer Indexing System](#chainindexer-indexing-system)
+- [Appendix](#appendix)
+  - [Wny isn't there an automated migration from the previous indexing system to the ChainIndexer indexing system?](#wny-isnt-there-an-automated-migration-from-the-previous-indexing-system-to-the-chainindexer-indexing-system)
+  - [`ChainValidateIndex` RPC API](#chainvalidateindex-rpc-api)
+
+## Introduction
+
+This document is for externally-available, high-performance RPC providers and for node operators who use or expose the Ethereum and/or events APIs.  It walks through the configuration changes, migration flow and operations/maintenance work needed to enable, backfill and maintain the [`ChainIndexer`](#chainindexer-indexing-system).  The justification for and benefits of the `ChainIndexer` are documented [here](https://github.com/filecoin-project/lotus/issues/12453). 
+
+The ChainIndexer is now also required if you enable Ethereum (`eth_*`) APIs using the `EnableEthRPC` Lotus configuration option.
+
```

Comment:
In addition -> `ChainIndexer` is ALSO required if user enables the ActorEventAPIs using the `EnableActorEventsAPI` using the Lotus configuration option

Reply:
Fixed in https://github.com/filecoin-project/lotus/pull/12600/commits/8463a03345d8c28410825d0cc115224976522d31.  Resolving.